ClangTidy warnings for `misc-include-cleaner`

```
/Users/jviotti/Projects/core/src/core/json/include/sourcemeta/core/json.h:8:1: error: included header json_error.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/json_error.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 | #include <sourcemeta/core/json_value.h>

/Users/jviotti/Projects/core/src/core/json/include/sourcemeta/core/json_value.h:9:1: error: included header json_hash.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    9 | #include <sourcemeta/core/json_hash.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   10 | #include <sourcemeta/core/json_object.h>

/Users/jviotti/Projects/core/src/core/json/include/sourcemeta/core/json_value.h:10:1: error: included header json_object.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   10 | #include <sourcemeta/core/json_object.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   11 |

/Users/jviotti/Projects/core/src/core/json/json.cc:5:1: error: included header parser.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    5 | #include "parser.h"
      | ^~~~~~~~~~~~~~~~~~~
    6 | #include "stringify.h"

/Users/jviotti/Projects/core/src/core/json/json.cc:6:1: error: included header stringify.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    6 | #include "stringify.h"
      | ^~~~~~~~~~~~~~~~~~~~~~
    7 |

/Users/jviotti/Projects/core/src/core/json/json_value.cc:1:1: error: included header json_array.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/json_array.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    2 | #include <sourcemeta/core/json_value.h>

/Users/jviotti/Projects/core/src/core/json/parser.h:7:1: error: included header grammar.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    7 | #include "grammar.h"
      | ^~~~~~~~~~~~~~~~~~~~
    8 |

/Users/jviotti/Projects/core/src/core/json/stringify.h:6:1: error: included header grammar.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    6 | #include "grammar.h"
      | ^~~~~~~~~~~~~~~~~~~~
    7 |

/Users/jviotti/Projects/core/src/core/jsonl/include/sourcemeta/core/jsonl.h:8:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 |

/Users/jviotti/Projects/core/src/core/jsonl/iterator.cc:1:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    2 | #include <sourcemeta/core/json_error.h>

/Users/jviotti/Projects/core/src/core/jsonl/iterator.cc:6:1: error: included header grammar.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    6 | #include "grammar.h"
      | ^~~~~~~~~~~~~~~~~~~~
    7 |

/Users/jviotti/Projects/core/src/core/jsonl/jsonl.cc:1:1: error: included header json_value.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/json_value.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    2 | #include <sourcemeta/core/jsonl.h>

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer.h:8:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 | #include <sourcemeta/core/uri.h>

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer.h:11:1: error: included header jsonpointer_error.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   11 | #include <sourcemeta/core/jsonpointer_error.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   12 | #include <sourcemeta/core/jsonpointer_pointer.h>

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer.h:12:1: error: included header jsonpointer_pointer.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   12 | #include <sourcemeta/core/jsonpointer_pointer.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   13 | #include <sourcemeta/core/jsonpointer_position.h>

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer.h:13:1: error: included header jsonpointer_position.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   13 | #include <sourcemeta/core/jsonpointer_position.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   14 | #include <sourcemeta/core/jsonpointer_subpointer_walker.h>

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer.h:14:1: error: included header jsonpointer_subpointer_walker.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   14 | #include <sourcemeta/core/jsonpointer_subpointer_walker.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   15 | #include <sourcemeta/core/jsonpointer_template.h>

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer.h:15:1: error: included header jsonpointer_template.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   15 | #include <sourcemeta/core/jsonpointer_template.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   16 | #include <sourcemeta/core/jsonpointer_walker.h>

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer.h:16:1: error: included header jsonpointer_walker.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   16 | #include <sourcemeta/core/jsonpointer_walker.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   17 |

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer.h:73:65: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    4 | auto get(const JSON &document, const Pointer &pointer) -> const JSON &;
      |                                                                 ^

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer_error.h:8:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 |

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer_error.h:19:14: error: no header providing "sourcemeta::core::JSONParseError" is directly included [misc-include-cleaner,-warnings-as-errors]
    4 | #ifndef SOURCEMETA_CORE_JSONPOINTER_EXPORT
    5 | #include <sourcemeta/core/jsonpointer_export.h>
    6 | #endif
    7 |
    8 | #include <sourcemeta/core/json.h>
    9 |
   10 | #include <cstdint> // std::uint64_t
   11 |
   12 | namespace sourcemeta::core {
   13 |
   14 | /// @ingroup jsonpointer
   15 | /// This class represents a parsing error.
   16 | class SOURCEMETA_CORE_JSONPOINTER_EXPORT PointerParseError
   17 |     // TODO: It makes no sense for a JSON Pointer error to inherit from a JSON
   18 |     // error. Make them independent
   19 |     : public JSONParseError {
      |              ^

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer_position.h:8:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 |

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer_position.h:10:1: error: included header jsonpointer_pointer.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   10 | #include <sourcemeta/core/jsonpointer_pointer.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   11 |

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer_position.h:51:25: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    4 |   auto operator()(const JSON::ParsePhase phase, const JSON::Type,
      |                         ^

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer_template.h:4:1: error: included header regex.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    4 | #include <sourcemeta/core/regex.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    5 |

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer_token.h:4:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    4 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    5 |

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer_token.h:13:17: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    4 | #include <sourcemeta/core/json.h>
    5 |
    6 | #include <cassert> // assert
    7 |
    8 | namespace sourcemeta::core {
    9 |
   10 | /// @ingroup jsonpointer
   11 | template <typename PropertyT, typename Hash> class GenericToken {
   12 | public:
   13 |   using Value = JSON;
      |                 ^

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer_walker.h:4:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    4 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    5 |

/Users/jviotti/Projects/core/src/core/jsonpointer/include/sourcemeta/core/jsonpointer_walker.h:19:30: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    4 | #include <sourcemeta/core/json.h>
    5 |
    6 | #include <cstddef> // std::size_t
    7 | #include <vector>  // std::vector
    8 |
    9 | namespace sourcemeta::core {
   10 |
   11 | /// @ingroup jsonpointer
   12 | /// A walker to get every JSON Pointer in a JSON document. Note that no specific
   13 | /// ordering is guaranteed. If you expect any ordering, sort afterwards.
   14 | template <typename PointerT> class GenericPointerWalker {
   15 | private:
   16 |   using internal = typename std::vector<PointerT>;
   17 |
   18 | public:
   19 |   GenericPointerWalker(const JSON &document) { this->walk(document, {}); }
      |                              ^

/Users/jviotti/Projects/core/src/core/jsonpointer/jsonpointer.cc:1:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    2 | #include <sourcemeta/core/json_hash.h>

/Users/jviotti/Projects/core/src/core/jsonpointer/jsonpointer.cc:4:1: error: included header jsonpointer.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    4 | #include <sourcemeta/core/jsonpointer.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    5 | #include <sourcemeta/core/jsonpointer_pointer.h>

/Users/jviotti/Projects/core/src/core/jsonpointer/jsonpointer.cc:6:1: error: included header uri.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    6 | #include <sourcemeta/core/uri.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    7 |

/Users/jviotti/Projects/core/src/core/jsonpointer/jsonpointer.cc:8:1: error: included header grammar.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include "grammar.h"
      | ^~~~~~~~~~~~~~~~~~~~
    9 | #include "parser.h"

/Users/jviotti/Projects/core/src/core/jsonpointer/jsonpointer.cc:9:1: error: included header parser.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    9 | #include "parser.h"
      | ^~~~~~~~~~~~~~~~~~~
   10 | #include "stringify.h"

/Users/jviotti/Projects/core/src/core/jsonpointer/jsonpointer.cc:10:1: error: included header stringify.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   10 | #include "stringify.h"
      | ^~~~~~~~~~~~~~~~~~~~~~
   11 |

/Users/jviotti/Projects/core/src/core/jsonpointer/parser.h:4:1: error: included header grammar.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    4 | #include "grammar.h"
      | ^~~~~~~~~~~~~~~~~~~~
    5 |

/Users/jviotti/Projects/core/src/core/jsonpointer/parser.h:6:1: error: included header json_value.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    6 | #include <sourcemeta/core/json_value.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    7 | #include <sourcemeta/core/jsonpointer.h>

/Users/jviotti/Projects/core/src/core/jsonpointer/parser.h:7:1: error: included header jsonpointer.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    7 | #include <sourcemeta/core/jsonpointer.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    8 | #include <sourcemeta/core/jsonpointer_error.h>

/Users/jviotti/Projects/core/src/core/jsonpointer/parser.h:8:1: error: included header jsonpointer_error.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/jsonpointer_error.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 |

/Users/jviotti/Projects/core/src/core/jsonpointer/stringify.h:4:1: error: included header json_value.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    4 | #include <sourcemeta/core/json_value.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    5 | #include <sourcemeta/core/uri.h>

/Users/jviotti/Projects/core/src/core/jsonpointer/stringify.h:5:1: error: included header uri.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    5 | #include <sourcemeta/core/uri.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    6 |

/Users/jviotti/Projects/core/src/core/jsonschema/bundle.cc:1:1: error: included header jsonschema.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/jsonschema.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    2 |

/Users/jviotti/Projects/core/src/core/jsonschema/bundle.cc:9:37: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/jsonschema.h>
    2 |
    3 | #include <cassert> // assert
    4 | #include <sstream> // std::ostringstream
    5 | #include <utility> // std::move
    6 |
    7 | namespace {
    8 |
    9 | auto embed_schema(sourcemeta::core::JSON &root,
      |                                     ^

/Users/jviotti/Projects/core/src/core/jsonschema/bundle.cc:50:38: error: no header providing "sourcemeta::core::SchemaFrame" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |                    sourcemeta::core::SchemaFrame &frame,
      |                                      ^

/Users/jviotti/Projects/core/src/core/jsonschema/frame.cc:1:1: error: included header jsonschema.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/jsonschema.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    2 |

/Users/jviotti/Projects/core/src/core/jsonschema/frame.cc:15:50: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/jsonschema.h>
    2 |
    3 | #include <algorithm>  // std::sort, std::all_of, std::any_of
    4 | #include <cassert>    // assert
    5 | #include <functional> // std::less
    6 | #include <map>        // std::map
    7 | #include <optional>   // std::optional
    8 | #include <set>        // std::set
    9 | #include <sstream>    // std::ostringstream
   10 | #include <utility>    // std::pair, std::move
   11 | #include <vector>     // std::vector
   12 |
   13 | enum class AnchorType : std::uint8_t { Static, Dynamic, All };
   14 |
   15 | static auto find_anchors(const sourcemeta::core::JSON &schema,
      |                                                  ^

/Users/jviotti/Projects/core/src/core/jsonschema/frame.cc:74:31: error: no header providing "sourcemeta::core::URI" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |       const sourcemeta::core::URI identifier(schema.at("$id").to_string());
      |                               ^

/Users/jviotti/Projects/core/src/core/jsonschema/frame.cc:231:29: error: no header providing "sourcemeta::core::SchemaFrame" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |     const sourcemeta::core::SchemaFrame &frame,
      |                             ^

/Users/jviotti/Projects/core/src/core/jsonschema/frame.cc:443:33: error: no header providing "sourcemeta::core::SchemaReferenceType" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |     if (location.first.first == SchemaReferenceType::Static) {
      |                                 ^

/Users/jviotti/Projects/core/src/core/jsonschema/frame.cc:456:25: error: no header providing "sourcemeta::core::stringify" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |       sourcemeta::core::stringify(location.second.pointer, stream);
      |                         ^

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema.h:8:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 | #include <sourcemeta/core/jsonpointer.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema.h:9:1: error: included header jsonpointer.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    9 | #include <sourcemeta/core/jsonpointer.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   10 |

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema.h:11:1: error: included header jsonschema_error.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   11 | #include <sourcemeta/core/jsonschema_error.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   12 | #include <sourcemeta/core/jsonschema_frame.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema.h:12:1: error: included header jsonschema_frame.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   12 | #include <sourcemeta/core/jsonschema_frame.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   13 | #include <sourcemeta/core/jsonschema_resolver.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema.h:13:1: error: included header jsonschema_resolver.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   13 | #include <sourcemeta/core/jsonschema_resolver.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   14 | #include <sourcemeta/core/jsonschema_transform.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema.h:14:1: error: included header jsonschema_transform.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   14 | #include <sourcemeta/core/jsonschema_transform.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   15 | #include <sourcemeta/core/jsonschema_types.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema.h:16:1: error: included header jsonschema_walker.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   16 | #include <sourcemeta/core/jsonschema_walker.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   17 |

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema.h:103:22: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    4 | auto is_schema(const JSON &schema) -> bool;
      |                      ^

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema.h:443:58: error: no header providing "sourcemeta::core::URI" is directly included [misc-include-cleaner,-warnings-as-errors]
    4 | auto reference_visitor_relativize(JSON &subschema, const URI &base,
      |                                                          ^

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_error.h:8:1: error: included header jsonpointer.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/jsonpointer.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 | #include <sourcemeta/core/uri.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_error.h:9:1: error: included header uri.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    9 | #include <sourcemeta/core/uri.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   10 |

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_frame.h:8:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 | #include <sourcemeta/core/jsonpointer.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_frame.h:9:1: error: included header jsonpointer.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    9 | #include <sourcemeta/core/jsonpointer.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   10 |

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_frame.h:11:1: error: included header jsonschema_resolver.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   11 | #include <sourcemeta/core/jsonschema_resolver.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   12 | #include <sourcemeta/core/jsonschema_types.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_frame.h:12:1: error: included header jsonschema_types.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   12 | #include <sourcemeta/core/jsonschema_types.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   13 | #include <sourcemeta/core/jsonschema_walker.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_frame.h:13:1: error: included header jsonschema_walker.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   13 | #include <sourcemeta/core/jsonschema_walker.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   14 |

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_frame.h:186:27: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    4 |   auto to_json() const -> JSON;
      |                           ^

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_resolver.h:8:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 | #include <sourcemeta/core/jsonschema_types.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_resolver.h:9:1: error: included header jsonschema_types.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    9 | #include <sourcemeta/core/jsonschema_types.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   10 |

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_resolver.h:54:18: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    4 |   auto add(const JSON &schema,
      |                  ^

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_transform.h:8:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 | #include <sourcemeta/core/jsonpointer.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_transform.h:9:1: error: included header jsonpointer.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    9 | #include <sourcemeta/core/jsonpointer.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   10 | #include <sourcemeta/core/uri.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_transform.h:10:1: error: included header uri.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   10 | #include <sourcemeta/core/uri.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   11 |

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_transform.h:12:1: error: included header jsonschema_resolver.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   12 | #include <sourcemeta/core/jsonschema_resolver.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   13 |

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_transform.h:90:14: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    4 |   auto apply(JSON &schema, const JSON &root, const Vocabularies &vocabularies,
      |              ^

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_types.h:4:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    4 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    5 | #include <sourcemeta/core/jsonpointer.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_types.h:5:1: error: included header jsonpointer.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    5 | #include <sourcemeta/core/jsonpointer.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    6 | #include <sourcemeta/core/uri.h>

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_types.h:6:1: error: included header uri.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    6 | #include <sourcemeta/core/uri.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    7 |

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_walker.h:8:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 |

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_walker.h:10:1: error: included header jsonschema_types.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   10 | #include <sourcemeta/core/jsonschema_types.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   11 |

/Users/jviotti/Projects/core/src/core/jsonschema/include/sourcemeta/core/jsonschema_walker.h:65:13: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    4 |       const JSON &input, const SchemaWalker &walker,
      |             ^

/Users/jviotti/Projects/core/src/core/jsonschema/jsonschema.cc:12:58: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/jsonschema.h>
    2 |
    3 | #include <cassert>     // assert
    4 | #include <cstdint>     // std::uint64_t
    5 | #include <functional>  // std::less
    6 | #include <limits>      // std::numeric_limits
    7 | #include <numeric>     // std::accumulate
    8 | #include <sstream>     // std::ostringstream
    9 | #include <type_traits> // std::remove_reference_t
   10 | #include <utility>     // std::move
   11 |
   12 | auto sourcemeta::core::is_schema(const sourcemeta::core::JSON &schema) -> bool {
      |                                                          ^

/Users/jviotti/Projects/core/src/core/jsonschema/jsonschema.cc:55:27: error: no header providing "sourcemeta::core::SchemaError" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |   throw sourcemeta::core::SchemaError(error.str());
      |                           ^

/Users/jviotti/Projects/core/src/core/jsonschema/jsonschema.cc:60:11: error: no header providing "sourcemeta::core::SchemaIdentificationStrategy" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |     const SchemaIdentificationStrategy strategy,
      |           ^

/Users/jviotti/Projects/core/src/core/jsonschema/jsonschema.cc:71:18: error: no header providing "sourcemeta::core::SchemaResolutionError" is directly included [misc-include-cleaner,-warnings-as-errors]
   71 |   } catch (const SchemaResolutionError &) {
      |                  ^

/Users/jviotti/Projects/core/src/core/jsonschema/jsonschema.cc:570:21: error: no header providing "sourcemeta::core::SchemaFrame" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |   sourcemeta::core::SchemaFrame frame{
      |                     ^

/Users/jviotti/Projects/core/src/core/jsonschema/jsonschema.cc:611:64: error: no header providing "sourcemeta::core::URI" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |     sourcemeta::core::JSON &subschema, const sourcemeta::core::URI &base,
      |                                                                ^

/Users/jviotti/Projects/core/src/core/jsonschema/official_resolver.in.cc:1:1: error: included header jsonschema.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/jsonschema.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    2 |

/Users/jviotti/Projects/core/src/core/jsonschema/official_walker.cc:1:1: error: included header jsonschema.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/jsonschema.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    2 |

/Users/jviotti/Projects/core/src/core/jsonschema/official_walker.cc:6:26: error: no header providing "sourcemeta::core::SchemaWalkerResult" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/jsonschema.h>
    2 |
    3 | auto sourcemeta::core::schema_official_walker(
    4 |     std::string_view keyword,
    5 |     const sourcemeta::core::Vocabularies &vocabularies)
    6 |     -> sourcemeta::core::SchemaWalkerResult {
      |                          ^

/Users/jviotti/Projects/core/src/core/jsonschema/official_walker.cc:9:31: error: no header providing "sourcemeta::core::SchemaKeywordType" is directly included [misc-include-cleaner,-warnings-as-errors]
    9 |     return {sourcemeta::core::SchemaKeywordType::strategy,                     \
      |                               ^

/Users/jviotti/Projects/core/src/core/jsonschema/resolver.cc:10:1: error: no header providing "sourcemeta::core::SchemaMapResolver" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/jsonschema.h>
    2 |
    3 | #include <algorithm> // std::transform
    4 | #include <cassert>   // assert
    5 | #include <cctype>    // std::tolower
    6 | #include <sstream>   // std::ostringstream
    7 |
    8 | namespace sourcemeta::core {
    9 |
   10 | SchemaMapResolver::SchemaMapResolver() {}
      | ^

/Users/jviotti/Projects/core/src/core/jsonschema/resolver.cc:16:11: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/jsonschema.h>
    2 |
    3 | #include <algorithm> // std::transform
    4 | #include <cassert>   // assert
    5 | #include <cctype>    // std::tolower
    6 | #include <sstream>   // std::ostringstream
    7 |
    8 | namespace sourcemeta::core {
    9 |
   10 | SchemaMapResolver::SchemaMapResolver() {}
   11 |
   12 | SchemaMapResolver::SchemaMapResolver(const SchemaResolver &resolver)
   13 |     : default_resolver{resolver} {}
   14 |
   15 | auto SchemaMapResolver::add(
   16 |     const JSON &schema, const std::optional<std::string> &default_dialect,
      |           ^

/Users/jviotti/Projects/core/src/core/jsonschema/resolver.cc:23:3: error: no header providing "sourcemeta::core::SchemaFrame" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |   SchemaFrame frame{SchemaFrame::Mode::References};
      |   ^

/Users/jviotti/Projects/core/src/core/jsonschema/resolver.cc:87:1: error: no header providing "sourcemeta::core::SchemaFlatFileResolver" is directly included [misc-include-cleaner,-warnings-as-errors]

/Users/jviotti/Projects/core/src/core/jsonschema/resolver.cc:110:22: error: no header providing "sourcemeta::core::SchemaIdentificationStrategy" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |       schema, *this, SchemaIdentificationStrategy::Loose, default_dialect,
      |                      ^

/Users/jviotti/Projects/core/src/core/jsonschema/resolver.cc:115:11: error: no header providing "sourcemeta::core::SchemaError" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |     throw SchemaError(error.str());
      |           ^

/Users/jviotti/Projects/core/src/core/jsonschema/transformer.cc:1:1: error: included header jsonschema.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/jsonschema.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    2 |

/Users/jviotti/Projects/core/src/core/jsonschema/transformer.cc:27:1: error: no header providing "sourcemeta::core::SchemaTransformRule" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 | SchemaTransformRule::SchemaTransformRule(std::string &&name,
      | ^

/Users/jviotti/Projects/core/src/core/jsonschema/transformer.cc:44:37: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 | auto SchemaTransformRule::transform(JSON &) const -> void {
      |                                     ^

/Users/jviotti/Projects/core/src/core/jsonschema/transformer.cc:51:9: error: no header providing "sourcemeta::core::SchemaReferenceError" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |   throw SchemaReferenceError(reference, origin,
      |         ^

/Users/jviotti/Projects/core/src/core/jsonschema/transformer.cc:59:39: error: no header providing "sourcemeta::core::SchemaFrame" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |                                 const SchemaFrame &frame,
      |                                       ^

/Users/jviotti/Projects/core/src/core/jsonschema/transformer.cc:70:18: error: no header providing "sourcemeta::core::SchemaAbortError" is directly included [misc-include-cleaner,-warnings-as-errors]
   70 |   } catch (const SchemaAbortError &) {
      |                  ^

/Users/jviotti/Projects/core/src/core/jsonschema/transformer.cc:99:6: error: no header providing "sourcemeta::core::SchemaTransformer" is directly included [misc-include-cleaner,-warnings-as-errors]
   99 | auto SchemaTransformer::check(
      |      ^

/Users/jviotti/Projects/core/src/core/jsonschema/walker.cc:14:35: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 | #include <sourcemeta/core/jsonschema.h>
    2 |
    3 | #include <algorithm> // std::max, std::sort
    4 | #include <cassert>   // assert
    5 |
    6 | namespace {
    7 | enum class SchemaWalkerType_t : std::uint8_t { Deep, Flat };
    8 |
    9 | auto walk(const std::optional<sourcemeta::core::Pointer> &parent,
   10 |           const sourcemeta::core::Pointer &pointer,
   11 |           const sourcemeta::core::PointerTemplate &instance_location,
   12 |           const sourcemeta::core::PointerTemplate &relative_instance_location,
   13 |           std::vector<sourcemeta::core::SchemaIteratorEntry> &subschemas,
   14 |           const sourcemeta::core::JSON &subschema,
      |                                   ^

/Users/jviotti/Projects/core/src/core/jsonschema/walker.cc:42:25: error: no header providing "sourcemeta::core::SchemaIdentificationStrategy" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |       sourcemeta::core::SchemaIdentificationStrategy::Strict,
      |                         ^

/Users/jviotti/Projects/core/src/core/jsonschema/walker.cc:57:23: error: no header providing "sourcemeta::core::SchemaIteratorEntry" is directly included [misc-include-cleaner,-warnings-as-errors]
   57 |     sourcemeta::core::SchemaIteratorEntry entry{parent,
      |                       ^

/Users/jviotti/Projects/core/src/core/jsonschema/walker.cc:398:19: error: no header providing "sourcemeta::core::SchemaIterator" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 | sourcemeta::core::SchemaIterator::SchemaIterator(
      |                   ^

/Users/jviotti/Projects/core/src/core/jsonschema/walker.cc:426:19: error: no header providing "sourcemeta::core::SchemaIteratorFlat" is directly included [misc-include-cleaner,-warnings-as-errors]
  426 | sourcemeta::core::SchemaIteratorFlat::SchemaIteratorFlat(
      |                   ^

/Users/jviotti/Projects/core/src/core/jsonschema/walker.cc:445:19: error: no header providing "sourcemeta::core::SchemaKeywordIterator" is directly included [misc-include-cleaner,-warnings-as-errors]
  445 | sourcemeta::core::SchemaKeywordIterator::SchemaKeywordIterator(
      |                   ^

/Users/jviotti/Projects/core/src/core/uri/escaping.cc:27:20: error: no header providing "URI_FALSE" is directly included [misc-include-cleaner,-warnings-as-errors]
   27 |                    URI_FALSE, URI_FALSE);
      |                    ^

/Users/jviotti/Projects/core/src/core/uri/escaping.cc:57:48: error: no header providing "URI_BR_DONT_TOUCH" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |       uriUnescapeInPlaceExA(buffer, URI_FALSE, URI_BR_DONT_TOUCH);
      |                                                ^

/Users/jviotti/Projects/core/src/core/uri/include/sourcemeta/core/uri.h:8:1: error: included header uri_error.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/uri_error.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 |

/Users/jviotti/Projects/core/src/core/uri/uri.cc:17:35: error: no header providing "URI_SUCCESS" is directly included [misc-include-cleaner,-warnings-as-errors]
   17 |   if (uriNormalizeSyntaxA(uri) != URI_SUCCESS) {
      |                                   ^

/Users/jviotti/Projects/core/src/core/uri/uri.cc:53:10: error: no header providing "URI_ERROR_SYNTAX" is directly included [misc-include-cleaner,-warnings-as-errors]
   53 |     case URI_ERROR_SYNTAX:
      |          ^

/Users/jviotti/Projects/core/src/core/uri/uri.cc:55:31: error: no header providing "sourcemeta::core::URIParseError" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |       throw sourcemeta::core::URIParseError{
      |                               ^

/Users/jviotti/Projects/core/src/core/uri/uri.cc:57:10: error: no header providing "URI_ERROR_MALLOC" is directly included [misc-include-cleaner,-warnings-as-errors]
   57 |     case URI_ERROR_MALLOC:
      |          ^

/Users/jviotti/Projects/core/src/core/uri/uri.cc:59:10: error: no header providing "URI_ERROR_OUTPUT_TOO_LARGE" is directly included [misc-include-cleaner,-warnings-as-errors]
   59 |     case URI_ERROR_OUTPUT_TOO_LARGE:
      |          ^

/Users/jviotti/Projects/core/src/core/uri/uri.cc:480:49: error: no header providing "URI_RESOLVE_STRICTLY" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 |                            &base.internal->uri, URI_RESOLVE_STRICTLY)) {
      |                                                 ^

/Users/jviotti/Projects/core/src/core/uri/uri.cc:483:10: error: no header providing "URI_ERROR_ADDBASE_REL_BASE" is directly included [misc-include-cleaner,-warnings-as-errors]
  483 |     case URI_ERROR_ADDBASE_REL_BASE:
      |          ^

/Users/jviotti/Projects/core/src/core/uri/uri.cc:517:31: error: no header providing "URI_FALSE" is directly included [misc-include-cleaner,-warnings-as-errors]
  517 |                               URI_FALSE, nullptr)) {
      |                               ^

/Users/jviotti/Projects/core/src/core/yaml/include/sourcemeta/core/yaml.h:8:1: error: included header json.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    8 | #include <sourcemeta/core/json.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    9 |

/Users/jviotti/Projects/core/src/core/yaml/include/sourcemeta/core/yaml.h:10:1: error: included header yaml_error.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   10 | #include <sourcemeta/core/yaml_error.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   11 |

/Users/jviotti/Projects/core/src/core/yaml/include/sourcemeta/core/yaml.h:63:47: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    4 | auto parse_yaml(const JSON::String &input) -> JSON;
      |                                               ^

/Users/jviotti/Projects/core/src/core/yaml/yaml.cc:5:1: error: included header yaml.h is not used directly [misc-include-cleaner,-warnings-as-errors]
    5 | #include <sourcemeta/core/yaml.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    6 |

/Users/jviotti/Projects/core/src/core/yaml/yaml.cc:14:26: error: no header providing "sourcemeta::core::JSON" is directly included [misc-include-cleaner,-warnings-as-errors]
    2 | #include <yaml.h>
    3 |
    4 | #include <sourcemeta/core/json_error.h>
    5 | #include <sourcemeta/core/yaml.h>
    6 |
    7 | #include <sstream>     // std::ostringstream, std::istringstream
    8 | #include <string_view> // std::string_view
    9 |
   10 | // TODO: Perform parsing token by token using `yaml_parser_parse`,
   11 | // as that function also let us get line/column information on `yaml_event_t`
   12 | static auto yaml_node_to_json(yaml_node_t *const node,
   13 |                               yaml_document_t *const document)
   14 |     -> sourcemeta::core::JSON {
      |                          ^

/Users/jviotti/Projects/core/src/extension/alterschema/alterschema.cc:183:10: error: no header providing "sourcemeta::core::SchemaTransformer" is directly included [misc-include-cleaner,-warnings-as-errors]
    1 | auto add(SchemaTransformer &bundle, const AlterSchemaCategory category)
      |          ^

/Users/jviotti/Projects/core/src/extension/alterschema/include/sourcemeta/core/alterschema.h:17:1: error: included header jsonschema.h is not used directly [misc-include-cleaner,-warnings-as-errors]
   17 | #include <sourcemeta/core/jsonschema.h>
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   18 |

/Users/jviotti/Projects/core/src/extension/alterschema/include/sourcemeta/core/alterschema.h:84:10: error: no header providing "sourcemeta::core::SchemaTransformer" is directly included [misc-include-cleaner,-warnings-as-errors]
   13 | auto add(SchemaTransformer &bundle, const AlterSchemaCategory category) -> void;
      |          ^

```

--- a/cmake/common/targets/clang-tidy.config
+++ b/cmake/common/targets/clang-tidy.config
@@ -2,17 +2,16 @@
 # See https://clang.llvm.org/extra/clang-tidy/index.html
 # First disable all default checks (with -*)
 Checks: '-*,
-          misc-include-cleaner'
-          # bugprone-*,-bugprone-branch-clone,-bugprone-easily-swappable-parameters,-bugprone-empty-catch,
-          # clang-analyzer-*,
-          # clang-diagnostic-*,
-          # modernize-*,
-          # concurrency-*,
-          # cppcoreguidelines-*,
-          # performance-*,
-          # portability-*,
-          # objc-*,
-          # misc-*,-misc-no-recursion'
+          bugprone-*,-bugprone-branch-clone,-bugprone-easily-swappable-parameters,-bugprone-empty-catch,
+          clang-analyzer-*,
+          clang-diagnostic-*,
+          modernize-*,
+          concurrency-*,
+          cppcoreguidelines-*,
+          performance-*,
+          portability-*,
+          objc-*,
+          misc-*,-misc-no-recursion'
 WarningsAsErrors: '*'
 HeaderFilterRegex: ''
 FormatStyle: none

